index.html

```diff
@@ -62,5 +62,184 @@ <h1>Hi👋, I'm Cephas Tokoli</h1>
+                    <div class="tech-stack">
+                        <p>HTML</p>
+                        <p>Ruby on Rails</p>
+                        <p>Javascript</p>
+                    </div>
+                    <div class="details-button">
+                       <div>
+                        <p >See this project</p>
+                        <img src="images/Union.svg" alt="union">
+                       </div>
+                    </div>
+                    </div>
+                </div>
+            </div>
+    
+            <div class="project">
+                <div class="laptop-img">
+                    <img src="images/laptop5.png" alt="laptop">
+                </div>
+                <div class="project-details">
+                    <div class="tech-details">
+                        <p class="project-name">Project name goes here</p>
+                    <div class="tech-stack">
+                        <p>HTML</p>
+                        <p>Ruby on Rails</p>
+                        <p>Javascript</p>
+                    </div>
+                    <div class="details-button">
+                       <div>
+                        <p >See this project</p>
+                        <img src="images/Union.svg" alt="union">
+                       </div>
+                    </div>
+                    </div>
+                </div>
+            </div>
+    </section>
+    <section class="about">
+        <div>
+            <h2 class="headings">About</h2>
+            <div class="content">
+               <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+               <button class="button download">Get CV
+                <div class="icon"></div>
+               </button>
+            </div>
+        </div>
+    </section>
+    <section class="competency">
+        <div class="grid-container">
+            <div class="language grid-item">
+                <h3>Languages</h3>
+                <ul>
+                    <li>Javascript(ES5/ES6)</li>
+                    <li>Ruby</li>
+                    <li>CSS</li>
+                    <li>HTML</li>
+                </ul>
+            </div>
+            <div class="frameworks grid-item">
+                <h3>Frameworks</h3>
+                <ul>
+                    <li>React.js</li>
+                    <li>Ruby on rails</li>
+                    <li>RSpec</li>
+                    <li>Capibara</li>
+                </ul>
+            </div>
+            <div class="skills grid-item">
+                <h3>Skills</h3>
+                <ul>
+                    <li>Database Management</li>
+                    <li>CLI</li>
+                    <li>API Design</li>
+                    <li>Version Control</li>
+                    <li>Web Development</li>
+                </ul>
+            </div>
+        </div>
+    </section>
```

Comment:
- [ ] Since it is an about section, I think it would be better to enclose both of them in a `section` tag and please make sure to align the content of the About section with `flexbox` ✨💪